src/components/Graphs/Graphs.jsx

```diff
@@ -0,0 +1,94 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import {StyledBody, StyledTitle} from 'baseui/card';
+import {useStyletron} from 'baseui';
+import groupBy from "lodash.groupby"
+
+import {StyledCard} from '..';
+import {useData} from '../../contexts/DataContext';
+import {CartesianGrid, Legend, Line, LineChart, Tooltip, XAxis, YAxis} from "recharts"
+import Loader from "../DataElement/Loader"
+import {sum} from "../../helpers/misc"
+import {Button} from "baseui/button"
+
+const CASES_KEY = 'cases';
+const DEATHS_KEY = 'deaths';
+const CURES_KEY = 'cures';
+
+function accumulateData(data) {
+  let cumulativeCases = 0;
+  let cumulativeDeaths = 0;
+  let cumulativeCures = 0;
+
+  return Object
+    .entries(groupBy(data, 'date'))
+    .map(([date, dataPerDate]) => {
+      const casesDateCount = sum(dataPerDate.filter(el => el.key === CASES_KEY))
+      const deathsDateCount = sum(dataPerDate.filter(el => el.key === DEATHS_KEY))
+      const curesDateCount = sum(dataPerDate.filter(el => el.key === CURES_KEY))
+
+      cumulativeCases += casesDateCount;
+      cumulativeDeaths += deathsDateCount;
+      cumulativeCures += curesDateCount;
+
+      return {
+        date: date.substring(5), // strip year
```

Comment:
I know it's hacky 🙈  ... but daylight saving time is coming and I didn't want to run into issues with parsing date and extracting month properly. Do you think it's ok?

Reply:
There is `moment.js` library installed you can use to parse date.

Reply:
Oh! That's great. I didn't want to add to the bundle size for such a simple use case but if it's already there then I'm gonna use it :)